src/game/client/components/chat.cpp

```diff
@@ -46,33 +46,6 @@ CChat::~CChat()
 
 void CChat::OnReset()
 {
-	for(int i = 0; i < MAX_LINES; i++)
```

Comment:
When you shuffle in the demo viewer the chat won't reset probably if you clear this function?

Reply:
You are right. I changed my commit. I tested switching between demos, offline, online and mapchange. I think it should be fine now.